apps/meteor/lib/callbacks.ts

```diff
@@ -118,7 +119,7 @@ type ChainedCallbackSignatures = {
 	) => Promise<T>;
 
 	'livechat.beforeRouteChat': (inquiry: ILivechatInquiryRecord, agent?: { agentId: string; username: string }) => ILivechatInquiryRecord;
-	'livechat.checkDefaultAgentOnNewRoom': (agent: SelectedAgent, visitor?: ILivechatVisitor) => SelectedAgent | null;
+	'livechat.checkDefaultAgentOnNewRoom': (agent: SelectedAgent, visitor?: ILivechatContactVisitorAssociation) => SelectedAgent | null;
```

Comment:
This one is no longer a visitor isn't it?

Reply:
We call the visitor association just `visitor` in a bunch of places, but I agree it's better to have a more descriptive name here. I'll change it

Reply:
Actually, I didn't need this param anymore :partying_face: